spec/at-rules/import.md

```diff
@@ -15,16 +15,25 @@ still supported for backwards-compatibility.
 
 <x><pre>
 **ImportRule**                ::= '@import' ImportArgument (',' ImportArgument)*
-**ImportArgument**            ::= ImportUrl ImportLayer? ImportSupportsDeclaration? [MediaQueryList][]?
-**ImportLayer**                 ::= 'layer' ('(' InterpolatedIdentifier ('.' InterpolatedIdentifier)* ')')?
+**ImportArgument**            ::= ImportUrl ImportModifier*
+**ImportModifier**            ::= ImportFunction | ImportSupports | [MediaQueryList]
+**ImportSupports**            ::= 'supports(' SupportsDeclaration ')'
+**ImportFunction**            ::= [InterpolatedIdentifier]¹ '(' InterpolatedDeclarationValue? ')'
 **ImportUrl**                 ::= QuotedString | [InterpolatedUrl][]
-**ImportSupportsDeclaration** ::= 'supports(' SupportsDeclaration ')'
 </pre></x>
 
 [InterpolatedIdentifier]: ../syntax.md#InterpolatedIdentifier
 [InterpolatedUrl]: ../syntax.md#InterpolatedUrl
 [MediaQueryList]: media.md#syntax
 
+1: This identifier may not be the `"supports"`. No whitespace is allowed between
+   it and the following `(`.
+
+> Note that this parses `@import "..." layer` differently than the CSS standard:
+> in CSS, `layer` is a CSS layering keyword but Sass parses it as part of a
+> media query. This doesn't pose a problem in practice because Sass's semantics
+> never depend on how import modifiers are parsed.
```

Comment:
this will be an issue if dart-sass keeps parsing the media query list to produce the CSS AST, because `layer screen` will not be a valid media query list, and would throw in that parsing.

Reply:
That's a problem for future me :smile: